src/payment/offsite-payment-mappers/payload-mapper.js

```diff
@@ -72,6 +72,7 @@ export default class PayloadMapper {
         const payload = objectAssign(
             {
                 amount: order.grandTotal ? order.grandTotal.integerAmount : null,
+                tax: order.taxTotal ? order.taxTotal.amount : null,
```

Comment:
Why are we sending this format of tax rather than the `integerAmount`?

Reply:
The correct format is `integerAmount`. I have changed this.